dev/com.ibm.ws.webcontainer.security/src/com/ibm/websphere/security/web/PasswordExpiredException.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.websphere.security.web;
+
+import javax.servlet.ServletException;
+/**
```

Comment:
Since this is going to be API, customers will be able to see the code (or at least, javadoc will get generated from this).  We need to say something about what this exception is used for.  We could say something like "This exception is thrown when a programmatic login is performed against a user registry which is configured to report when a user's password is expired."

We should say something similar for the user revoked exception (the one that extends ServletException).